shiny/BrownDog/server.R

```diff
@@ -0,0 +1,140 @@
+library(shiny)
+library(leaflet)
+library(RPostgreSQL)
+library(PEcAn.DB)
+
+
+# Define server logic
+server <- shinyServer(function(input, output, session) {
+  output$agreementUI <- renderUI({
+    if (is.null(input$type)){
+      return()
+    }
+    
+    # Depending on input$type, we'll generate a different lincense agreement
+    switch(input$type,
+           "AmeriFlux" =  checkboxInput("agreement", paste0("I agree to AmeriFlux license."), value = FALSE, width = NULL),
+           "NARR" =  checkboxInput("agreement", paste0("I agree to NARR license."), value = FALSE, width = NULL),
+           "FLUXNET" =  checkboxInput("agreement", paste0("I agree to FLUXNET license."), value = FALSE, width = NULL)
+    )
+  })
+  
+  observeEvent(input$type, {
+    #get all sites name, lat and lon by sitegroups
+    dbparams <- list(user = "bety", dbname = "bety", password = "bety", host = "localhost")
```

Comment:
done. I also add web/config.php by copying web/config.example.php in softwareserver-0002.